diagnostic.md

```diff
@@ -6,38 +6,38 @@ Write your answers inside this file, where it's indicated by the comments.
     What command do we use to do that? Use the correct URL for your fork of this
     repository in your answer.
 
-    ```sh
-    <!-- Remove this comment and place your answer here. -->
+    ```
+    git clone https://github.com/tina513/git-basics-diagnostic.git
```

Comment:
`git clone git@github.com/tina513/git-basics-diagnostic.git`. the SSH url for **YOUR** forked repository.